.gitignore

```diff
@@ -8,6 +8,14 @@
 # Test binary, built with `go test -c`
 *.test
 
+# Ignore the server binary output by `go build`
+server
+
```

Comment:
seems unnecessary to use server as even in docs we don't mention use server. Similarly keploy-logs are deleted after keploy is finished running.

Reply:
The reason being that during development, it is upto the dev to chose either ```go build```, in which case the server binary executable gets made. It would be a nice edition to .gitignore to improve the developer experience.

As also mentioned in #1445 , it means it is a requested feature update for the developers

Keploy-logs are not deleted if there was an error during execution if i am not wrong. Hence again, it improves the developer experience by adding it to the gitignore

Reply:
1. The issue seems to have occurred once and it could be avoided by cross-checking the changes which seems is a standard practice.
2. Keploy logs are generated in the project folder where keploy is run to record or test not in the base repo.

Though, to make it more simpler for developer this can be added in `.gitignore`